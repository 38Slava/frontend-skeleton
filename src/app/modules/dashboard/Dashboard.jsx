import React from 'react'

export default class Dashboard extends React.Component {

  render() {
    return <div id='dash'><h2>DASHBOARD</h2></div>
  }
}
