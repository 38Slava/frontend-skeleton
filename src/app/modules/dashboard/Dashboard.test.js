// Link.react-test.js
import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from './Dashboard';
import renderer from 'react-test-renderer';

describe('components', function() {
  describe('<Dashboard />', function() {
    it('renders correctly', function() {
      var dashboard = renderer.create(<Dashboard />).toJSON();

      expect(dashboard).toMatchSnapshot();
      expect(dashboard.props.id).toEqual('dash')
    });

    it('should contain correct text inside', function() {
      const wrapper = shallow(<Dashboard />);
      const welcome = <h2>DASHBOARD</h2>;

      expect(wrapper.contains(welcome)).toEqual(true);
    });
  });
});
