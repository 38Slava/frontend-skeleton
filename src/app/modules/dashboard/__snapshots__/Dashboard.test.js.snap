// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <Dashboard /> renders correctly 1`] = `
<div
  id="dash"
>
  <h2>
    DASHBOARD
  </h2>
</div>
`;
